cmd/swctl/app/cmd_dependency.go

```diff
@@ -491,6 +491,14 @@ func InstallDocker(cli Cli, dockerVersion string) error {
 		color.Fprintln(cli.Out(), out)
 
 	}
+	errorGroup := postInstall(cli)
```

Comment:
PostInstall errors should fail the installation because failing adding current user to docker group will result in failure later.